fedbiomed/common/constants.py

```diff
@@ -260,12 +248,10 @@ class SecaggElementTypes(_BaseEnum):
 
     Attributes:
         SERVER_KEY: server key split between the parties
-        BIPRIME: biprime shared between the parties
         DIFFIE_HELLMAN: one pair of DH key for each node party, public key shared with other node parties
     """
     SERVER_KEY: int = 0
-    BIPRIME: int = 1
-    DIFFIE_HELLMAN: int = 2
+    DIFFIE_HELLMAN: int = 1
```

Comment:
As you renumbered the `SecaggElementTypes` you need to increase the major version of messaging protocol and secagg element

```
__messaging_protocol_version__ = FBM_Component_Version('5')  # format of gRPC messages.
__secagg_element_version__ = FBM_Component_Version('3')  # format os secagg database elements
```

With current code, a node before this PR would accept to negotiate secagg with a researcher with this PR (or vice versa), but would mistake a biprime secagg negotiation for a DH negotiation (or refuse to do a DH nego etc.)

Reply:
Hi @mvesin,

Thanks for catching it. Then, I am not going to change the element type value. It will avoid changing the protocol version.